payment/src/main/resources/db/changelog/ddl/changelog-0002.sql

```diff
@@ -0,0 +1,10 @@
+
+create table payment_provider (
+   id                           varchar(255) not null,
+   is_enabled                   boolean,
+   name                         varchar(255),
+   configure_url                varchar(255),
+   landing_view_component_name  varchar(255),
+   additional_settings          varchar(255),
```

Comment:
I think you should change the type of this field to text because we can't determine how long the content will be.

Reply:
thanks